Gateway/Http/Util/HttpUtil.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace DigitalHub\Ebanx\Gateway\Http\Util;
+
+class HttpUtil {
+	public static function setupEbanxClient($config, $credit_card_config)
```

Comment:
Add type in this params

Reply:
With parameters type specified, i can't nullify the creditCardConfigs, as we decided to do.